qiita_db/commands.py

```diff
@@ -242,3 +246,52 @@ def load_processed_data_cmd(fps, fp_types, processed_params_table_name,
     return ProcessedData.create(processed_params_table_name,
                                 processed_params_id, list(zip(fps, fp_types)),
                                 preprocessed_data, study, processed_date)
+
+
+def load_parameters_from_cmd(name, fp, table):
+    """Add a new parameters entry on `table`
+
+    Parameters
+    ----------
+    fp : str
+        The filepath to the parameters file
+    table : str
+        The name of the table to add the parameters
+
+    Returns
+    -------
+    qiita_db.BaseParameters
+        The newly `qiita_db.BaseParameters` object
+
+    Raises
+    ------
+    ValueError
+        If the table does not exists on the DB
+        If the fp is not correctly formatted
```

Comment:
Would be good to have a notes section explaining the format of the parameters file.